MetaMask/metamask-extension
# Nft warning modal

## Explanation



This PR will add a new warning slide-up modal when the user approves the collection of NFTs.


## More Information



Fixes #15700 
Fixes #15701 

## Screenshots/Screencaps



### Before



It is a new modal.

### After



https://user-images.githubusercontent.com/97883527/194866843-32e09401-f3c7-4194-983e-7fb03eb17dd1.mov



## Manual Testing Steps



Open the test dapp https://metamask.github.io/test-dapp/
Under the collectibles section, click on "Deploy"
Wait for transaction to be confirmed and click on "Mint"
Wait for transaction to be confirmed and click on "Set Approval For All"
Click Approve button
Notice there is a new nft warning modal displayed with slide up animation

## Pre-Merge Checklist


### + If there are functional changes:
